Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoCortexDataLake.yaml

```diff
@@ -0,0 +1,176 @@
+          DvcId = DeviceExternalID,
+          DvcOriginalAction = DeviceAction,
+          EventOriginalSeverity = LogSeverity,
+          EventOriginalType = DeviceEventClassID,
+          EventOriginalUid = ExtID,
+          EventProductVersion = DeviceVersion,
+          HttpContentFormat = RequestContext,
+          HttpReferrer = PanOSReferer,
+          RuleName = DeviceCustomString1,
+          SrcDvcId = PanOSSourceUUID,
+          SrcMacAddr = PanOSSourceDeviceMac,
+          SrcNatIpAddr = SourceTranslatedAddress,
+          SrcNatPortNumber = SourceTranslatedPort,
+          SrcPortNumber = SourcePort,
+          SrcUsername = SourceUserName,
+          SrcZone = DeviceCustomString4,
+          Url = RequestURL,
+          UrlCategory = DeviceCustomString2
+      | extend
+          Dst = coalesce(DstDvcId, DstHostname, DstIpAddr),
+          Dvc = coalesce(DvcId, DvcHostname, DvcIpAddr),
+          EventEndTime = EventStartTime,
+          HttpStatusCode = EventResultDetails,
+          Src = coalesce(SrcDvcId, SrcHostname, SrcIpAddr),
+          NetworkProtocolVersion = case(
+                              DstIpAddr contains ".",
+                              "IPv4",       
+                              DstIpAddr contains ":",
+                              "IPv6", 
+                              ""
+                          ),
+          NetworkDirection = iff(PanOSIsClienttoServer=="true","Outbound", "Inbound"),
+          Rule = RuleName,
+          SrcUserType = _ASIM_GetUserType(SrcUsername, ""),
+          DstUserType = _ASIM_GetUserType(DstUsername, ""),
+          User = SrcUsername,
+          Hostname = DstHostname,
+          IpAddr = SrcIpAddr,
+          SessionId = NetworkSessionId,
+          DvcIdType = iff(isnotempty(DvcId), "Other", ""),
+          SrcDvcIdType = iff(isnotempty(SrcDvcId), "Other", ""),
+          DstDvcIdType = iff(isnotempty(DstDvcId), "Other", ""),
+          SrcDomainType = iff(isnotempty(SourceNTDomain), "Windows", SrcDomainType),
+          DstDomainType = iff(isnotempty(DestinationNTDomain), "Windows", DstDomainType),
+          SrcUsernameType = _ASIM_GetUsernameType(SrcUsername),
+          DstUsernameType = _ASIM_GetUsernameType(DstUsername)
+      | extend
+          EventSchema = "WebSession",
+          EventSchemaVersion = "0.2.6",
+          EventType = "HTTPsession"
+      | project-away
+          Source*,
+          Destination*,
+          Device*,
+          AdditionalExtensions,
+          CommunicationDirection,
+          EventOutcome,
+          PanOS*,
+          Protocol,
+          ExternalID,
+          Message,
+          start,
+          EndTime,
+          Activity,
+          FieldDevice*,
+          Flex*,
+          File*,
+          Old*,
+          MaliciousIP*,
+          OriginalLogSeverity,
+          Process*,
+          ReceivedBytes,
+          SentBytes,
+          Remote*,
+          Request*,
+          SimplifiedDeviceAction,
```

Comment:
what's the difference between DeviceAction and SimplifiedDeviceAction?

Reply:
DeviceAction field contains the value coming from source whereas SimplifiedDevcieAction field contains simplified value of DeviceAction.